src/libtest/lib.rs

```diff
@@ -1258,8 +1258,9 @@ pub mod bench {
         let ns_iter_summ = bs.auto_bench(f);
 
         let ns_iter = cmp::max(ns_iter_summ.median as u64, 1);
-        let iter_s = 1_000_000_000 / ns_iter;
-        let mb_s = (bs.bytes * iter_s) / 1_000_000;
+        // use iterations per 100s for better resolution in mb_s.
+        let iter_per_100s = 100 * 1_000_000_000 / ns_iter;
+        let mb_s = (bs.bytes * iter_per_100s) / 100 / 1_000_000;
```

Comment:
Indeed, that seems good. I don't think we have to worry about overflow, no reason not to do this.